src/SIM/AdaptiveSetup.h

```diff
@@ -54,11 +54,13 @@ class AdaptiveSetup
   //! \param[in] iStep Refinement step counter
   //! \param[in] gNorm Global norms
   //! \param[in] refIn Element refinement indicators (element error norms)
+  //! \param[in] currDofs Current number of model DOFs. -1 to use model.getNoDOFs()
```

Comment:
Use SIMbase::getNoDOFs() instead in the doxy, and the hyperlink is automatically created.

Reply:
done.